test/cmd/test-service/main.go

```diff
@@ -29,10 +29,6 @@ const (
 	EnvServiceCall = "SERVICE_CALL"
 )
 
-var (
```

Comment:
Why removed?

Reply:
My super IDE told me it's unused :)

Reply:
Hench the comment on it.. ;)

Reply:
LOL :) Speedy move - didnt even notice the comment :) Brought back in 5a02456

But on that note - can we document how and when are using "dev mode"? Can't really see any reference in the project. Maybe elaborate the comment will be sufficient and most accurate.

Reply:
Does it matter if you don't read? :P hehe

Can't even remember atm. It's not in the Makefile. There is a flag you provide to go-bindata, but I must have been playing with it outside of Make when I was developing the UI there.

Reply:
I did search everywhere before removing.... So if you don't remember what's the value of keeping it around? :D

Reply:
As I said, it's related to go-bindata and I know how that work. I just don'r remember how I used it back in the day. It's needed to use the debug feature of go-bindata, how ever you generate the go-bindata

Reply:
Restored in 5a024564